pubnub-core/pubnub-core-impl/src/main/kotlin/com/pubnub/internal/PubNubCore.kt

```diff
@@ -112,16 +112,8 @@ class PubNubCore internal constructor(
     val configuration: BasePNConfiguration,
     val listenerManager: ListenerManager,
     eventEnginesConf: EventEnginesConf = EventEnginesConf(),
+    private val pnsdkName: String = "PubNub-JVM",
```

Comment:
Why there is default value here?

Reply:
I thought it might be useful to create PNCore in tests, but turns out I didn't use it. Removing